### PR TITLE
List User Followers

### PR DESCRIPTION
**Pull request by a7madM**
**Load More**
Screenshot from 2020-04-17 20-00-29

**List All Followers**
Screenshot from 2020-04-17 19-59-43

No Followers

Screenshot from 2020-04-17 19-59-58

### CODE REVIEW

File: src/app/views/followerships/_followers.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<%= render followers, followers: @followers %>
```
[the remark this answers is not in the document]
Comment (a7madM) [reply src/app/views/followerships/_followers.html.erb:1]: we need it for some style customization

File: src/app/views/followerships/_followership.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= render 'users/user', user: followership.follower %>
```
Comment (emad-elsaid) [inline src/app/views/followerships/_followership.html.erb:1]: we can't render followership.follower directly?
```suggestion
<%= render followership.follower %>
```

File: src/app/views/users/_user.html.erb [hunk 3]
```diff
@@ -15,8 +15,9 @@
             @<%= user.username %>
           <% end %>
         </p>
-
-        <%= render "users/follow", current_user: current_user, user: user %>
+        <% if show_actions?  %>
+          <%= render "users/actions", user: user %>
```
Comment (emad-elsaid) [inline src/app/views/users/_user.html.erb:19]: we can render this file outside of the user partial, in the comments/index and other pages that render the user partial

Comment (a7madM) [reply src/app/views/users/_user.html.erb:19]: it will break the design !

File: src/app/views/followerships/_followership.html.erb (hunk 2 above)
Comment (a7madM) [reply src/app/views/followerships/_followership.html.erb:1]: done

File: src/app/controllers/followerships_controller.rb [hunk 4]
```diff
@@ -1,8 +1,19 @@
 class FollowershipsController < ApplicationController
+  layout 'users'
+
   before_action :set_user
   before_action :set_followership, only: %i[destroy]
+  before_action :set_followership, only: %i[destroy]
```
Comment (emad-elsaid) [inline src/app/controllers/followerships_controller.rb:6]: this is a redundant line

**Pull request merged**